bundles/org.openhab.binding.mqtt.espmilighthub/README.md

```diff
@@ -152,6 +154,14 @@ For example:
 | 0xB4CA    | 8        | 0xB4CA8  |
 | 0xB4CA    | 0        | 0xB4CA0  |
 
+## Using the ALL group
+
+The ALL group with the Group ID 0 can be used to control all bulbs that are paired with one specific remote at once. While this functionality can also be achieved by using OpenHab groups with even greater flexibility the ALL group must be setup if you want to capture remote events for the ALL group to keep physical devices synchronized to their OpenHab representations. Milight remotes send all commands with the Group ID 0 after the master ON/OFF buttons have been used. If the ALL group has not been setup these events will be lost and your Item states will no longer be synchonized with the actual device states until you issue a command via OpenHab. If you do not use a remote at all or you only control other bulbs than the ones controlled by OpenHab you should not need to setup the ALL group.
```

Comment:
A few things.

1. Please use `openHAB` note it starts with lowercase o, and HAB is uppercase.
2. Place each sentence on a new line by pressing enter after each full stop. There are at least 5 sentences on this one line number 159.

Reply:
Done